CelesteNet.Client/Entities/PicoGhost.cs

```diff
@@ -0,0 +1,130 @@
+using System;
+using System.Collections.Generic;
+using Celeste.Mod.CelesteNet.Client.Components;
+using Celeste.Mod.CelesteNet.DataTypes;
+using Celeste.Pico8;
+using Microsoft.Xna.Framework;
+using Monocle;
+using MonoMod.Utils;
+
+namespace Celeste.Mod.CelesteNet.Client.Entities;
+
+public class FontHelper {
```

Comment:
Shouldn't this `FontHelper` be in its own file rather than in PicoGhost.cs?... And just in the `Celeste.Mod.CelesteNet.Client` namespace, not under `Entities`